account_index_based_currency/models/account_invoice.py

```diff
@@ -93,6 +88,7 @@ def default_get(self, default_fields):
         copy=False, digits=(12, 6))
     agreement_currency_rate = fields.Float(
         help="Currency rate this Document was agreed",
+        default=lambda *args: 1,
```

Comment:
If it is not needed recomputed it since that is a static value then you can use just `default=1`

Reply:
Done